base/logging.go

```diff
@@ -421,6 +91,12 @@ func Tracef(logKey LogKey, format string, args ...interface{}) {
 	logTo(LevelTrace, logKey, format, args...)
 }
 
+func Statsf(stats []byte) {
```

Comment:
I know it's nitpicky, but the signature of this function isn't similar to the other logging functions like Infof, Warnf, Errorf, the f denoting it takes string formatting and args.

Should this named something different to avoid confusion? `RecordStats` maybe?

Reply:
Agreed.  Renamed to `RecordStats`.